src/components/languageMenu/LanguageMenuPure.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import OutsideClickHandler from 'react-outside-click-handler'
+import LanguageContext from 'src/contexts/language'
+import './language-menu.less'
+import configPages from 'src/config'
+
+const LanguageMenuPure = ({
+  closeLanguageMenu,
+  toggleLanguageMenu,
+  isLanguageMenuShown
+}) => {
```

Comment:
Could you please make a destructive inside the function if it's doesn't fit a single row. 
(props) => {  
  const { one, two, three } = props 
}

Reply:
done